CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Changelog
 
-## 0.1.0 (13.08.2021)
+## 0.1.1 (05.11.2021)
+* Prepare library
```

Comment:
This messages is ambiguous. "Prepare library", for?

Reply:
I'm testing the draft pull requests workflow, I will add the full info in a while.